src/js/components/settings/organization.js

```diff
@@ -40,6 +42,7 @@ export default class MyOrganization extends React.Component {
     }, 5000);
   }
   render() {
+    const currentPlan = 'Mender Enterprise';
```

Comment:
This looks like it will always be set as Mender Enterprise, unless I'm reading it wrong? Tried testing with the following env. variables
```
  hasMultitenancy: "true",
  isEnterprise: "",
  isHosted: "true"
```

Reply:
Thanks for pointing that out and I changed the settings to rely on the Hosted/ Enterprise values.
Those values were more set as a placeholder until we determined where that information might come from, but until then using the environment should be enough! 👍